inst/WORDLIST

```diff
@@ -112,6 +112,7 @@ displayUnit
 docx
 dplyr
 dv
+eg

```

Comment:
it should be **e.g.**
let's remove **eg** from the wordlist and fix the function description instead